README.md

```diff
@@ -25,6 +25,15 @@ Rules:
        alt="Sponsored by Evil Martians" width="236" height="54">
 </a>
 
+## Pre-Commit Hook
+
+This project is also available as a pre-commit hook. Please add the following to your pre-commit configuration.
```

Comment:
Let’s add link here. `pre-commit` is a popular name, there are many projects with this name. People will now understand it without a link.